activation/activation_test.go

```diff
@@ -350,36 +350,6 @@ func TestBuilder_PublishActivationTx_FailsWhenNipstBuilderFails(t *testing.T) {
 	r.False(published)
 }
 
-func TestBuilder_PublishActivationTx_ActiveSetSizeMismatchPanics(t *testing.T) {
```

Comment:
Do we have a test that checks CalcActiveSetSize?, also, if the fix youre introducing might be temporary I'd comment it instead of deleting, to make sure we renable the test when enabling functionality

Reply:
https://github.com/spacemeshos/go-spacemesh/blob/de1fc2fd4fbcf218add9502d54a4185d3630156b/activation/activationdb_test.go#L107-L165

Yeah, I thought about commenting out instead of deleting too: https://github.com/spacemeshos/go-spacemesh/pull/1217#issuecomment-515788982 -- will do.

Reply:
Restored the deleted code and commented it out ✅